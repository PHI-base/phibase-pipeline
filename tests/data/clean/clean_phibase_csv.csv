Curation_comments_temp,_to_do_,Record ID,PHI_MolConn_ID,IdentifierTypeOfProteinID,ProteinID,IdentifierTypeOfGeneLocusID,GeneLocusID,AA sequence #no EMBL#,NT sequence #no EMBL#,Genomic sequence providing strain,Gene_name,Genome location,Specific modification/s to the targeted protein or promoter,Accession ID for the modified genetic element,Known interacting protein(s) in the pathogen,Interacting protein - locus ID,Multiple_mutation,Pathogen_NCBI_species_Taxonomy ID,Pathogen_species,Pathogen_NCBI_strain_Taxonomy_ID,Experimental_strain,Disease_name,Host_descripton,Host_NCBI_Taxonomy_ID,Experimental_host_species,Host_strain_genotype_or cultivar_taxonomy_ID_NCBI,HostGenotype_definedGene of interest,HostGenotype_definedGene of interest _AccessionID,tissue_type,Function,GO_annotation,Database,Pathway_secretion_systems,Phenotype_of_mutant,Mating_defect_prior_to_penetration,Pre_penetration_defect,Penetration_defect,Post_penetration_defect,Disease_development_macroscopically_visible,Vegetative_spores,Sexual_spores,In_vitro_growth,Spore_germination,Essential_gene_Lethal_knockout,Inducer,ChemicalAccession (Chebi/CAS),Tested Host_target,TestedHostTarget_AccessionID,Interaction phenotype,Host_response,Experimental_evidence,Transient Assay Experimental Evidence,Species_Expert,Entered_by,Literature_ID,Literature_source,DOI,Full_citation,Author_email,Comments,Reference ,Year_published,Curation details,File_name_pdf_files_provided,_batch_no_,Curation date,_curator_organisation_,__lab__,__FG_mycotoxin__,Additional IdentifierTypeOfGeneLocusID,Additional GeneLocusID,Anti-infective (Chemical),Compound,Target site,Group name,Chemical group,Mode in planta,Mode of action ,FRAC CODE,Additional comments  on anti-infectives
Curation Comments,To do,Record ID,PHI_MolConn_ID,Protein ID source,Protein ID,Gene ID source,Gene ID,AA sequence,NT sequence,Sequence Strain,Gene,Chr location,Gene/Protein modification,Modified gene/protein Id,Interacting partner(s),Interacting partner(s) Id,Multiple mutation,Pathogen ID,Pathogen species,Pathogen strain ID,Pathogen strain,Disease,Host description,Host ID,Host species,Host strain,Host genotype,Host genotype-Id,Tissue,Gene Function,GO annotation,Database,Pathway,Mutant Phenotype,Mating defect,Prepenetration defect,Penetration defect,Postpenetration defect,Disease manifestation,Vegetative spores,Sexual spores,Invitro growth,Spore germination,Essential gene,Gene inducer,Gene inducer Id,Host target,Host target Id,Interaction phenotype,Host response,Exp. Technique-stable,Exp. Technique-transient,Species expert,Entered by,PMID,Ref. Source,DOI,Ref. detail,Author email,Comments,Author reference,Year,Curation details,File name,Batch no.,Curation date,Curator organization,Lab,FG-mycotoxin,AdditionalIdentifierTypeOfGeneLocusID,AdditionalGeneLocusID,Anti infective agent,Anti infective compound,Anti infective-target site,Anti infective group name,Anti infective Chemical group,Anti infective-Mode in planta,Mode of action,FRAC CODE,Anti infective-comments
ﬀ,,Record 1,PHI:3,UniProt,P26215,EMBL,AAA79885,,,SB111,PGN1,no data found,,,,Uniprot:ABC,PHI:1;PHI:2,5017,Bipolaris zeicola,,SB111,leaf spot,monocots,4577,Zea mays,,,,colon epithelial cell,endopolygalacturonase,GO:0004650,GO,,unaffected pathogenicity,,no,no,no,,LOWERcase,LOWERcase,LOWERcase,LOWERcase,,pectin,,,,,,gene DISRUPTION,,,MU; JA,2152162,PubMed,10.1105/tpc.2.12.1191,,,Expression during all infection stages. pathogen formerly called Cochliobolus carbonum teleomorph name,Scott-Craig et al.,1990,,,PHI-base Vers.3.3,04/05/2005,Rres,,,,,,,,,,,,,
ﬃ,,Record 2,PHI:7,Uniprot,P22287,Genbank,CAA42824,,,race 5,AVR9,,,,,Genbank:ABC,PHI:1 PHI:2,,,,,"leaf mold, leaf spot",eudicots,4081,Solanum lycopersicum (related: tomato),,,,lung; Heart; kidney,effector protein,,,,effector (plant avirulence determinant),,,,,,,,,,,,,,,,,gene deletion; gene complementation; biochemical evidence,,,MU,1799694,Pubmed,no data found,,,,Van Kan et al.,1991,,,PHI-base Vers.3.3,39206,rres,,,,,,,,,,,,,
ﬄ,,Record 3,PHI:12,UniProt,Q01886,EMBL,  AAA33023,,,SB111,HTS1,,,,,,,5017,Bipolaris zeicola,,SB111,leaf spot,monocots,No host tests done,Zea mays (related: maize),,,,Peyer’s gland,cylic peptide synthase,"GO:0009405, IMP",GO,,loss of pathogenicity,Yes,Yes,Yes,Yes,,,,,,No,,,,,,,gene disruption,,,TKB; JA,11607305,pubmed,no data found,,,pathogen formerly called Cochliobolus carbonum teleomorph name,Panaccione et al.,1992,,,PHI-base Vers.3.3,16-Nov,RRES,,,,,,,,,,,,,
ﬂ ﬁ,,Record 4,PHI:14,UniProt,P0C017,EMBL,AAB09711,,,clinical isolate,ADE2,,,,,,,5207,Cryptococcus neoformans,235443,var. grubii serotype A strain H99,cryptococcosis,nematodes,Lethal pathogen phenotype,Caenorhabditis elegans (related: roundworm),,,,soft-tissue,phosphoribosylaminoimidazole carboxylase,"GO:0009405, IMP",GO,,reduced virulence,No,No,No,No,,,,,,Yes,,,,,,,gene disruption; gene complementation,,,TKB; KHK,,PubMed,10.1073/pnas.232568599,,,,Perfect et al.,,,,PHI-base Vers.3.3,May-19,MC/ Rres,,,,,,,,,,,,,