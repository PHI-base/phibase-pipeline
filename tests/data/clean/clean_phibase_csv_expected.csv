curation_comments,to_do,record_id,phi_molconn_id,protein_id_source,protein_id,gene_id_source,gene_id,aa_sequence,nt_sequence,sequence_strain,gene,chr_location,gene_protein_modification,modified_gene_protein_id,interacting_partners,interacting_partners_id,multiple_mutation,pathogen_id,pathogen_species,pathogen_strain_id,pathogen_strain,disease,host_description,host_id,host_species,host_strain,host_genotype,host_genotype_id,tissue,gene_function,go_annotation,database,pathway,mutant_phenotype,mating_defect,prepenetration_defect,penetration_defect,postpenetration_defect,disease_manifestation,vegetative_spores,sexual_spores,in_vitro_growth,spore_germination,essential_gene,gene_inducer,gene_inducer_id,host_target,host_target_id,interaction_phenotype,host_response,exp_technique_stable,exp_technique_transient,species_expert,entered_by,pmid,ref_source,doi,ref_detail,author_email,comments,author_reference,year,curation_details,file_name,batch_no,curation_date,curator_organization,lab,fg_mycotoxin,additionalidentifiertypeofgenelocusid,additionalgenelocusid,anti_infective_agent,anti_infective_compound,anti_infective_target_site,anti_infective_group_name,anti_infective_chemical_group,anti_infective_mode_in_planta,mode_of_action,frac_code,anti_infective_comments
ff,,Record 1,PHI:3,UniProt,P26215,EMBL,AAA79885,,,SB111,PGN1,,,,,UniProt:ABC,PHI:1; PHI:2,5017,Bipolaris zeicola,,SB111,leaf spot,monocots,4577,Zea mays,,,,colonic epithelial cell,endopolygalacturonase,GO:0004650,GO,,unaffected pathogenicity,,no,no,no,,LOWERcase,LOWERcase,LOWERcase,LOWERcase,,pectin,,,,,,gene disruption,,,MU; JA,2152162,PubMed,10.1105/tpc.2.12.1191,,,Expression during all infection stages. pathogen formerly called Cochliobolus carbonum teleomorph name,Scott-Craig et al.,1990,,,PHI-base Vers.3.3,2005-05-04,RRes,,,,,,,,,,,,,
ffi,,Record 2,PHI:7,UniProt,P22287,GenBank,CAA42824,,,race 5,AVR9,,,,,GenBank:ABC,PHI:1; PHI:2,,,,,leaf mold; leaf spot,eudicots,4081,Solanum lycopersicum,,,,lung; heart; kidney,effector protein,,,,effector (plant avirulence determinant),,,,,,,,,,,,,,,,,gene deletion; gene complementation; biochemical evidence,,,MU,1799694,PubMed,,,,,Van Kan et al.,1991,,,PHI-base Vers.3.3,2007-05-04,RRes,,,,,,,,,,,,,
ffl,,Record 3,PHI:12,UniProt,Q01886,EMBL,AAA33023,,,SB111,HTS1,,,,,,,5017,Bipolaris zeicola,,SB111,leaf spot,monocots,,Zea mays,,,,Peyer's gland,cylic peptide synthase,"GO:0009405, IMP",GO,,loss of pathogenicity,yes,yes,yes,yes,,,,,,no,,,,,,,gene disruption,,,TKB; JA,11607305,PubMed,,,,pathogen formerly called Cochliobolus carbonum teleomorph name,Panaccione et al.,1992,,,PHI-base Vers.3.3,2016-11-01,RRes,,,,,,,,,,,,,
fl fi,,Record 4,PHI:14,UniProt,P0C017,EMBL,AAB09711,,,clinical isolate,ADE2,,,,,,,5207,Cryptococcus neoformans,235443,var. grubii serotype A strain H99,cryptococcosis,nematodes,,Caenorhabditis elegans,,,,soft tissue,phosphoribosylaminoimidazole carboxylase,"GO:0009405, IMP",GO,,reduced virulence,no,no,no,no,,,,,,yes,,,,,,,gene disruption; gene complementation,,,TKB; KHK,,PubMed,10.1073/pnas.232568599,,,,Perfect et al.,,,,PHI-base Vers.3.3,2019-05-01,MC/RRes,,,,,,,,,,,,,
